pom.xml

```diff
@@ -1905,8 +1905,8 @@
             </dependency>
             <dependency>
                 <groupId>net.javacrumbs.json-unit</groupId>
-                <artifactId>json-unit-fluent</artifactId>
-                <version>1.5.5</version>
+                <artifactId>json-unit-assertj</artifactId>
+                <version>2.0.0.RC1</version>
```

Comment:
not a final version?

Reply:
yes, we spoke about that during our meeting about lib upgrades and my comment about why was written in the doc : there's no stable release of json-unit-assertj yet but as it allows to drop non-standard fluent API, I proposed to use it anyway.